pipelines/controller/src/main/java/com/google/fhir/analytics/PipelineManager.java

```diff
@@ -762,8 +765,8 @@ public void run() {
         manager.setLastRunDetails(currentDwhRoot, SUCCESS);
       } catch (Exception e) {
         logger.error("exception while running pipeline: ", e);
-        manager.captureError(fhirContext, currentDwhRoot, e);
-        manager.setLastRunDetails(currentDwhRoot, FAILURE);
+        manager.captureError(currentDwhRoot != null ? currentDwhRoot : "", e);
```

Comment:
Hmm, if `currentDwhRoot` is `null` where are we saving the error file? I suppose it would end up as a relative path which is probably not a good choice.